release/_releases_json.sh

```diff
@@ -1,6 +1,24 @@
 #!/bin/bash
 
+function _add_new_version {
+	lc_log INFO "Add ${LIFERAY_RELEASE_VERSION} version to supported-${LIFERAY_RELEASE_PRODUCT_NAME}-versions.txt"
+
+	if [ -n "${LIFERAY_RELEASE_PRODUCT_NAME}" ]
+	then
+		echo -en "\n${LIFERAY_RELEASE_VERSION}" >> "${_RELEASE_ROOT_DIR}"/supported-"${LIFERAY_RELEASE_PRODUCT_NAME}"-versions.txt
+	else
+		echo "The script reads the following environment variables:"
+		echo ""
+		echo "    LIFERAY_RELEASE_PRODUCT_NAME: Set to \"portal\" or \"dxp\" "
+		echo "    LIFERAY_RELEASE_VERSION: Set a version to add it in supported-{product}-versions.txt"
+		echo ""
+		echo "Example: LIFERAY_RELEASE_PRODUCT_NAME=dxp LIFERAY_RELEASE_VERSION=2024.q1 ${0}"
```

Comment:
@Wykenz, do you really need the variable `${0}`?

Reply:
Yes, in the help's format the `${0}` is the script, it will print out the example to the console like: `LIFERAY_RELEASE_PRODUCT_NAME=dxp LIFERAY_RELEASE_VERSION=2024.q1 ./rebuild_releases_json.sh`